community/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v2_3/TransactionBoundQueryContext.scala

```diff
@@ -202,18 +203,18 @@ final class TransactionBoundQueryContext(tc: TransactionalContextWrapper)
 
       case rangeLessThan: RangeLessThan[Number] =>
         rangeLessThan.limit(BY_NUMBER).map { limit =>
-          readOps.nodesGetFromIndexRangeSeekByNumber(index, null, false, limit.endPoint, limit.isInclusive)
+          readOps.indexQuery(index, IndexQuery.range(index.property, null, false, limit.endPoint, limit.isInclusive))
```

Comment:
Would it be easier to read if we had methods `rangeFrom(...)` and `rangeTo(...)` instead of passing null values to `range(...)`?